src/Schemes/AffineSchemes/Objects/Properties.jl

```diff
@@ -0,0 +1,487 @@
+export is_empty, issubset, is_open_embedding, is_closed_embedding
+
+
+
+####################################################################################
+# (1) Check if a scheme is empty
+####################################################################################
+
+@Markdown.doc """
+    isempty(X::AbsSpec)
+
+This method returns `true` if the affine scheme ``X`` is empty.
+Otherwise, `false` is returned.
+
+# Examples
+```jldoctest
+julia> X = affine_space(QQ,3)
+Spec of Multivariate Polynomial Ring in x1, x2, x3 over Rational Field
+
+julia> isempty(X)
+false
+
+julia> isempty(EmptyScheme(QQ))
```

Comment:
Another example could be a subscheme defined by the (1) ideal

Reply:
Thank you. I have added this.